src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -119,10 +119,9 @@ const NavHeader = (props) => {
         source={{uri: coverPath}}
         style={[styles.navContainer(isHeaderHide, topPosition), heightWithCoverImage()]}
         imageStyle={{opacity: isHeaderHide ? 0 : 1}}>
-        <Animated.Image
-          source={{uri: coverPath}}
-          style={[styles.headerImage(opacityHeaderAnimation), heightWithCoverImage()]}
-        />
+        <Animated.View style={styles.headerImage(opacityHeaderAnimation)}>
+          <FastImage source={{uri: coverPath}} style={heightWithCoverImage()} />
+        </Animated.View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `Animated.Image` to `Animated.View` wrapping a `FastImage` component is a good one. `FastImage` is known for its performance benefits over the standard React Native Image component, such as better memory usage and faster loading times. However, it's important to ensure that the `FastImage` library is properly installed and linked in your project, otherwise this could lead to runtime errors.

```diff
-        <Animated.Image
-          source={{uri: coverPath}}
-          style={[styles.headerImage(opacityHeaderAnimation), heightWithCoverImage()]}
-        />
+         <Animated.View style={styles.headerImage(opacityHeaderAnimation)}>
+           <FastImage source={{uri: coverPath}} style={heightWithCoverImage()} />
+         </Animated.View>
```
Please also make sure that the animations are still working as expected after this change, since `FastImage` might behave differently than the standard `Image` component when used with `Animated`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->